src/manifest-creation.ts

```diff
@@ -4,16 +4,16 @@ import path from "path";
 import updateDotenv from "update-dotenv";
 import { ProbotOctokit } from "./octokit/probot-octokit";
 import type { OctokitOptions } from "./types";
+import { loadPackageJson } from "./helpers/load-package-json";
 
 export class ManifestCreation {
+  #pkg: { [key: string]: any } | null = null;
```

Comment:
Why is this necessary?

Reply:
The typings?

Reply:
No, I mean why you added caching for the file. I don't think it's necessary.